tensorflow/core/kernels/depthtospace_op.cc

```diff
@@ -24,6 +24,7 @@ limitations under the License.
 #include "tensorflow/core/kernels/depthtospace_op.h"
 
 #include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+
```

Comment:
Small nit: Prefer to avoid making unrelated formatting changes.

Reply:
Thanks @josh11b. The unrelated formatting changes in those two files have been removed.